src/language.js

```diff
@@ -207,7 +207,7 @@ const createNewOptionModal = {
     <>
       Genus not found in MERMAID
       <br />
-      <a target="_blank" href="https://datamermaid.org/contact-us">
+      <a target="_blank" href="https://datamermaid.org/contact-us" rel="noreferrer">
```

Comment:
@alanjleonard , this answers your question about noreferrer you had a while back. Linter made me do it. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

Reply:
But we likely want to track where traffic is coming from. We own both domain and that link isn't dynamic. I don't see a reason for enforcing the `rel="noreferrer"`

Reply:
This is also a client-side security fix. If somebody was being malicious, they could just open devtools and remove that attribute.

Reply:
It talks about this in the `when to override it` section. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#when-to-override-it

Reply:
Ya, good point. Its pointing to our own content, and we definitely want to know how many people click on the cntact us page (vs who actually contact us. Could be an interesting metric if a contact us form had a bug or bad UX in theory)